resource-service/src/main/resources/application.yaml

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+spring:
+  profiles:
+    include:
+    - akka
+    - default-config
+
+---
 
 spring:
   application:
```

Comment:
This will still require specifying the second config after the `---` is called `default-config`
See other service yamls.

Reply:
Yep, added the profile to it.